src/main/java/frc/robot/commandGroup/PickUpCube.java

```diff
@@ -0,0 +1,22 @@
+package frc.robot.commandGroup;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.arm.commands.SetArmsPosition;
+import frc.robot.subsystems.gripper.Gripper;
+import frc.robot.subsystems.intake.Command.Feed;
+
+public class PickUpCube extends SequentialCommandGroup {
+    public PickUpCube(Gripper gripper, double intakePower, Translation2d armPosition) {
+        addCommands(
+                // Feeds the cube into the Intake until the BeamBreaker sends a false signal
+                new Feed(intakePower, 0),
```

Comment:
Testing in the future may show that an angle other than 0 should be used. The angle should be passed as a parameter or saved as a constant.

Reply:
done.